user/customizing-the-build.md

```diff
@@ -264,11 +264,9 @@ branches and tags that start with `deploy-` in any combination of cases.
 
 ## Skipping a build
 
-If you don't want to run a build for a particular commit, because all you are
-changing is the README for example, add `[ci skip]` to the git commit message.
+If you don't want to run a build for a particular commit, because all you are changing is the README for example, add `[ci skip]` or `[skip ci]` to the git commit message.
```

Comment:
Hmm...agreed - that should go! Was referencing notes on the phrasing and forgot to ditch that example!